src/admin/templates/dot-qserv/qserv.conf

```diff
@@ -0,0 +1,62 @@
+#
+# The Qserv client configuration file.
+#
+# Located in "$HOME/.lsst/qserv.conf" and used to run integration tests
+#
+# This file is used by Qserv client tools like :
+# - qserv-test-integration.py,
+# - qserv-check-integration.py (--help)
+#
+# For example :
+# qserv-check-integration --load --case=01
+# will :
+#   - load test dataset for case01 in both Qserv and MySQL
+#   - run a set of queries against both databases and compare results
```

Comment:
probably not? The commit comment could be clearer, but these are brought in as-is from qserv-operator. Since a lot of the schema will be moved from the templates folder into modules I think the remainder of this code would go away. At the very least it will need scrubbing for anything vestigial.